bitnami/argo-cd/templates/known-hosts-cm.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
```

Comment:
I'd allow existing cms

Reply:
It cannot be allowed here since the name must be the one it has for Argo CD to read it, so I added an optional flag to create it or not: `createExtraKnownHosts`. If you already have a config map with that name, just set that to `false` and your existing configmap will be taken